prime-router/src/main/kotlin/fhirengine/translation/hl7/utils/FhirPathUtils.kt

```diff
@@ -94,14 +94,15 @@ object FhirPathUtils : Logging {
     fun evaluateCondition(
```

Comment:
Function comment probably could use an update to help define the differences between the 3 similar params.